changelog.d/7688.bugfix

```diff
@@ -0,0 +1 @@
+Ensure the log context is kept during user interactive authentication. The bug was introduced in 1.13.0rc1.
```

Comment:
might be nice to rephrase this as a user-visible symptom. "Fix 'sentinel context' warning" or whatever.

Reply:
Ah, good call. 👍 

```suggestion
Fix "Starting db txn 'get_completed_ui_auth_stages' from sentinel context" warning. The bug was introduced in 1.13.0rc1.
```